### PR TITLE
CDAP-7736 Clarify Worker stop in Javadoc

### PR DESCRIPTION
**Pull request by Denton-L**
JIRA: https://issues.cask.co/browse/CDAP-7736

### CODE REVIEW

**Review by albertshau — APPROVED** [review]
lgtm

File: cdap-api/src/main/java/co/cask/cdap/api/worker/Worker.java [hunk 1]
```diff
@@ -53,7 +53,8 @@
   void destroy();
 
   /**
-   * Request to stop the running worker.
+   * Stop the {@code Worker}. This method will be invoked whenever the worker is externally stopped by CDAP.
+   *
    * This method will be invoked from a different thread than the one calling the {@link #run()} method.
```
Comment (awholegunch) [inline cdap-api/src/main/java/co/cask/cdap/api/worker/Worker.java:58]: I'm wondering about this link here: `{@link #run()}` : it is resolving to `Runnable.run()` or http://docs.oracle.com/javase/7/docs/api/java/lang/Runnable.html?is-external=true#run().

That could be correct, but I'm not sure. Could you find out (may need to ask @chtyim)?

**Comment by awholegunch** [comment]
One question, otherwise LGTM.

File: cdap-api/src/main/java/co/cask/cdap/api/worker/Worker.java (hunk 1 above)
Comment (Denton-L) [reply cdap-api/src/main/java/co/cask/cdap/api/worker/Worker.java:58]: I verified by building the docs locally and it works exactly as expected.

**Pull request merged**